sample/build.gradle

```diff
@@ -48,7 +48,10 @@ dependencies {
     implementation "com.squareup.okhttp3:logging-interceptor:4.10.0"
 
     // kevin. SDK
-    implementation "eu.kevin.android:core:2.6.4"
-    implementation "eu.kevin.android:accounts:2.6.4"
-    implementation "eu.kevin.android:in-app-payments:2.6.4"
+    // Import dependencies in your project like that:
```

Comment:
Yep, I remember that, and I find it pretty inconvenient when working on the new features, additionally there is some hustle in aligning versions in sample project and maven repo. For the sake of the sample app, I think it's totally alright to have project deps and some explanatory comment. However, if you have strong opinion on that, could rollback ofc.

Reply:
My initial idea was to update these versions after sdk is released

Reply:
Is there a real benefit of using real deps though?

Reply:
I guess I would confirm since this way it is clear that the sample uses the same latest thing that is currently released.

Reply:
It's a sample, so I'd say it makes sense to keep everything as close to what client would actually have to do. In this case, they would need to include actual deps instead of project module

Reply:
Alright, reverted to using real deps 👁️ 👄 👁️